x/bank/proto/cosmos/bank/v2/authorityMetadata.proto

```
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package cosmos.bank.v2;
+
+import "gogoproto/gogo.proto";
```

buf

import "gogoproto/gogo.proto": file does not exist

(COMPILE)